src/Sulu/Bundle/DocumentManagerBundle/Resources/config/core.xml

```diff
@@ -46,7 +46,7 @@
         </service>
 
         <service id="sulu_document_manager.slugifier" class="Symfony\Cmf\Api\Slugifier\CallbackSlugifier" public="false">
-            <argument>Ferrandini\Urlizer::urlize</argument>
+            <argument>Gedmo\Sluggable\Util\Urlizer::urlize</argument>
```

Comment:
The Gedmo shipped Urlizer is basically the same as the Ferrandini\Urlizer but as depend on https://github.com/Behat/Transliterator it will be updated soon when https://github.com/Behat/Transliterator/pull/29 is merged.